src/streamlink/stream/dash.py

```diff
@@ -0,0 +1,160 @@
+import itertools
+from streamlink import StreamError, PluginError
+from streamlink.compat import urlparse, urlunparse
+from streamlink.plugin.api import http
```

Comment:
**http** can be removed for `self.session`

Reply:
And this wasn't working... :)